김현수/mvvmstudy/app/src/main/java/com/example/mvvmstudy/IO/IGithubService.kt

```diff
@@ -0,0 +1,15 @@
+package com.example.mvvmstudy.IO
+
+import com.example.mvvmstudy.data.Repositories
+import retrofit2.Call
+import retrofit2.http.GET
+import retrofit2.http.Query
+
+interface IGithubService {
+    @GET("search/repositories")
+    fun searchRepositories(
+        @Query("q") query: String,
+        @Query("page") page:Int,
+        @Query("per_page") per_page:Int
```

Comment:
code format 하면 좋을 것 같아요!
```suggestion
        @Query("q") query: String,
        @Query("page") page: Int,
        @Query("per_page") per_page: Int
```